### PR TITLE
Updating instructions and screenshots

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9530

@robmazz please review/merge.

### CODE REVIEW

File: Teams/legal-hold.md [hunk 1]
```diff
@@ -38,26 +38,26 @@ To put a user or a team on legal hold in a eDiscovery (Standard) case:
...
 
    ![An image showing the Holds tab selected, and the Create button underneath.](media/LegalHold2.png)
 
    1. **Name your hold**. Select a descriptive and unique name for the hold you're going to create.
   
        ![This screenshot shows the Name your hold tab, where you can enter in a name and description for the hold you are creating.](media/LegalHold3.png)
 
-   2. **Choose location**. Choose whether you want the hold to be applied on a user or on an entire Team (a hold can't be applied on individual channels for now). If a user is on hold, all their messages are preserved, including messages in 1:1 chats, group chats, and private channels. Messages in standard and shared channels are preserved when the parent team is placed on hold.
+   2. **Choose locations**. Choose whether you want the hold to be applied on a user or on an entire Team (a hold can't be applied on individual channels for now). If a user is on hold, all their messages are preserved, including messages in 1:1 chats, group chats, and private channels. Messages in standard and shared channels are preserved when the parent team is placed on hold.
```
Comment (JohanFreelancer9) [inline Teams/legal-hold.md:56]:
```suggestion
   2. **Choose locations**. Choose whether you want the hold to be applied on a user or on an entire team (a hold can't be applied on individual channels for now). If a user is on hold, all their messages are preserved, including messages in 1:1 chats, group chats, and private channels. Messages in standard and shared channels are preserved when the parent team is placed on hold.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit, please.

**Comment by scanum** [comment]
@robmazz 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@robmazz @tiburd

**Comment by scanum** [comment]
@robmazz
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@tiburd Can you please help us here? thanks

**Comment by tiburd** [comment]
Confirmed menu item name changes in screenshots.
Merging.

**Pull request merged**